pom.xml

```diff
@@ -1031,6 +1031,12 @@
                 <artifactId>truth</artifactId>
                 <version>${truth.version}</version>
             </dependency>
+            <dependency>
```

Comment:
why add gson dependence ?

Reply:
> why add gson dependence ?

After importing  grpc-java 1.30.2, the ClientWorker class prompts that gson does not exist. need to import additional gson and set the scope .